sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -1268,7 +1268,7 @@ private[hive] object HiveClientImpl extends Logging {
   }
 }
 
-case object HiveVoidType extends DataType {
+private[hive] case object HiveVoidType extends DataType {
```

Comment:
hmmm, I thought `org.apache.spark.sql.hive` is already a private package, no? cc @viirya @dongjoon-hyun

Reply:
I think so.

Reply:
Nah, it's exposed. https://spark.apache.org/docs/3.0.1/api/java/org/apache/spark/sql/hive/package-summary.html. We should better explicitly make it private

Reply:
Got it, will fix it in the next commit.

Reply:
Thank you for pinging me. +1 for making it private.